doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+
+If you do not require customization, you can use the simpler using APDL commands to solve a harmonic analysis 
+with the Krylov method.
+ 
+The procedure described here implements an analysis identical to that defined by the macros.
+Details on the theory and equations describing the Krylov method can be found in the works of Puri [1]_ and Eser [2]_ .
+
+The exposure in PyMAPDL follows the same the same theory as the apdl macro and has the following methods:
+
+* :func:`KrylovSolver.krygensub() <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` Creates the Krylov subspace. 
+* :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` Solves the reduced system of equations.
+* :func:`KrylovSolver.kryexpand() <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` Expands the Krylov subspace.
+
+.. warning:: These methods must be run consecutively.
+
+Usage
+=====
+
+Generate the full file (``.full``) for Krylov method using Mechanical APDL:
+
+.. code:: py
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    
+    >>> mapdl = launch_mapdl()
+    >>> mapdl.prep7()
+
+	# Generate the FEA model (mesh, constraints, loads, etc.)
+  # ...
+
+    >>> mapdl.run("/SOLU")
+    >>> mapdl.antype("HARMIC")  # HARMONIC ANALYSIS
+    >>> mapdl.hropt("KRYLOV")
+    >>> mapdl.eqslv("SPARSE")
+    >>> mapdl.harfrq(0,1000)  # Set beginning and ending frequency
+    >>> mapdl.nsubst(100)  # Set the number of frequency increments
+    >>> mapdl.wrfull(1)  # GENERATE .FULL FILE AND STOP
+    >>> mapdl.solve()
+    >>> mapdl.finish()
+
+Create Instance of the Krylov Class
+-----------------------------------
+
+.. code:: py
+    
+    >>> mk = mapdl.krylov
+
+Call the :func:`krygensub <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` method which creates the Krylov subspace:
+Build a subspace of Size / Dimension 10 and at a frequency of 500 Hz.
+
+.. code:: py
+
+    >>> Qz = mk.krygensub(10, 500, True, True)
+
+Return the Krylov subspace
+--------------------------
+
+Call the :func:`krysolve <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` method which reduces system of equations and solve at each frequency:
+Solve from 0 Hz to 1000 Hz with 100 intervals in between, with stepped loading   
+
+.. code:: py
+
+    >>> Yz = mk.krysolve(0, 1000, 100, 1, True)
+
+
+Return the Reduced solution over Frequency range
+------------------------------------------------
+            
+Call the :func:`kryexpand <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` method which expand the reduced solution back to FE space:
+Output the expanded solution and calculate residual.   
+
+.. code:: py
+
+    >>> res = mk.kryexpand(True, 3)
+
+Return Ndarray (if out_key = True) solution vectors mapped to User order.
+
+.. note:: The Ndarray returned by the method ``kryexpand`` contains the node number along with dof solution,
+          for each of the calculated frequency.
```

Comment:
```suggestion
.. note:: The ``ndarray`` returned by the method ``kryexpand`` contains the node number along with the dof solution
          for each of the calculated frequencies.
```

Reply:
I think "dof" in line 120 needs to be defined (or at least I don't know what it means). It's possible your users understand "dof" though.